### ISSUE TITLE
Chi square Goodness of Fit

### ISSUE THREAD

**Issue by jessicaelaine24** [jasp-stats/jasp-desktop#1107]
In teaching JASP with undergraduate statistics classes, the one thing that I was completely unable to do with it was Chi Square Goodness of Fit. That would be an excellent addition!

**Comment by coledavis** [comment]
I would agree with this. It fits neatly between Binomial and Chi Square Test of Association. Binomial gives two conditions using frequencies, 1 x 1; CS G.o.F. gives a number of conditions, N x 1;. CST.o.A gives you a grid of conditions, N:N.

**Comment by boutinb** [comment]
We do have Chi Square test in the Contingency analysis. But it is maybe not what you mean? In that case, can you be more specific, and give an example of what you would like?

**Comment by coledavis** [comment]
The 'Chi Square test' of which you speak is the Chi Square test of Association, _not_ the Chi Square Goodness of Fit test. 

Example: 
There are 3 (or 4 or more) categories of observations. An example would be, 'Catholics', 'Protestants', 'Jews', 'Hindus' and 'Buddhists'.  A standard version of Goodness of Fit would see if the proportion of observations was different from random (i.e. .20, .20, .20, .20, .20). Instead, if a known proportion was the precedent (say, 10, 40, 10, 25, 15), then the test would ascertain if the current observations were significantly different from the precedent.
